choreolib/src/main/java/choreo/auto/AutoTrajectory.java

```diff
@@ -56,6 +56,45 @@ public class AutoTrajectory {
       Choreo.multiAlert(
           causes -> "Unable to get initial pose for trajectories " + causes + ".", kError);
 
+  /**
+   * Creates a trigger that produces a rising edge when any of the trajectories are finished.
+   *
+   * @param trajectories the trajectories to watch
+   * @return a trigger that determines if any of the trajectories are finished
+   */
+  public static Trigger anyDone(AutoTrajectory... trajectories) {
+    return anyDone(0, trajectories);
+  }
+
+  /**
+   * Creates a trigger that produces a rising edge when any of the trajectories are finished.
+   *
+   * @param cyclesToDelay The number of cycles to delay.
+   * @param trajectories the trajectories to watch
+   * @return a trigger that determines if any of the trajectories are finished
+   */
+  public static Trigger anyDone(int cyclesToDelay, AutoTrajectory... trajectories) {
+    var trigger = trajectories[0].done(cyclesToDelay);
```

Comment:
IIRC its legal to pass in no trajectories to a vararg, you can make the signature `(int cyclesToDelay, AutoTrajectory trajectory, AutoTrajectory... trajectories)`